CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Fixed: `function-calc-no-unspaced-operator` correctly interprets negative fractional numbers without leading zeros and those wrapped in parentheses.
 - Fixed: `number-max-precision` now ignores uppercase and mixed case `@import` at-rules.
 - Fixed:  Use `process.exitCode` with `stdOut` CLI to allow the process to exit naturally and avoid truncating output.
+- Fixed: `no-extra-semicolons` now ignores semicolons after Less mixins.
```

Comment:
https://github.com/stylelint/stylelint/pull/1599 updated postcss-less in package.json to fix the issue, but that PR didn't contain tests or a changelog item. So, I added them in this PR. I forgot to reference the previous PR though.